src/services/queries/Events.js

```diff
@@ -15,3 +15,17 @@ export const EVENTS_QUERY = gql`
     }
   }
 `;
+
+export const EVENT_BY_ID_QUERY = gql`
+  query GetEventById($id: uuid!) {
+    events(where: { id: { _eq: $id } }) {
```

Comment:
Maybe try event_by_pk instead 

The  you use an alias, like 

event: event_by_pk (...

Reply:
Of course, I'll try

Reply:
Thanks!